### ISSUE TITLE
add support for single precision fftw

### ISSUE THREAD

**Issue by GoogleCodeExporter** [votca/csg#104]
```
Many people use single precision gromacs and hence have single precision 
version of fftw, but not double precision one.

All Changes are in crosscorrelate.cc
-copy data to a new vector in case of single precision
-added the prefix of all fftw calls from fftw_XXX to fftwf_XXX
-add a cmake options
```

Original issue reported on code.google.com by `christop...@gmail.com` on 29 Jun 2011 at 1:59
- Blocked on: #105

**Comment by junghans** [comment]
FFTW in double precision is more common now, so closing as won't fix.
